Src/Jobbie.Sample.Client.WebApi.Host/Contracts/JobExecution.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Jobbie.Sample.Client.WebApi.Host.Contracts
+{
+    public sealed class JobExecution
+    {
+        public string Description { get; set; }
+        public string Cron { get; set; }
+
+        public override string ToString() => $"Description={Description}";
```

Comment:
It could be.  I just added this for debugging/testing purposes.  I'll include it now.